test/plugins/wasi_llm/wasi_llm.cpp

```diff
@@ -0,0 +1,55 @@
+#include "common/defines.h"
+#include "plugin/plugin.h"
+#include "runtime/callingframe.h"
+#include "runtime/instance/module.h"
+#include "types.h"
+#include "wasillmmodule.h"
+#include <algorithm>
+#include <array>
+#include <cstdint>
+#include <gtest/gtest.h>
+#include <string>
+#include <vector>
+
+using WasmEdge::Host::WASILLM::ErrNo;
+
+namespace {
+WasmEdge::Runtime::Instance::ModuleInstance *createModule() {
+  using namespace std::literals::string_view_literals;
+  WasmEdge::Plugin::Plugin::load(
```

Comment:
this is so weird, i tested in my local linux box, and I can't load this plugin successfully. any ideas about why?

Reply:
Do you receive any error log? Or could you check if the `libwasmedgePluginWasiLLM.so` is under such a folder?

Reply:
well i did some hacks into `load` and it says `no such file or directory` but I don't think so...
```
❯ cd build/test/plugins/wasi_llm
❯ pwd
/home/jun/Documents/WasmEdge/build/test/plugins/wasi_llm
❯ ls ../../../plugins/wasi_llm/libwasmedgePluginWasiLLM.so
../../../plugins/wasi_llm/libwasmedgePluginWasiLLM.so
❯ cd ../../../../
❯ build/test/plugins/wasi_llm/WasiLLMTests
[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from WasiLLMTest
[ RUN      ] WasiLLMTest.TrainGPT2
No such file or directory
/home/jun/Documents/WasmEdge/test/plugins/wasi_llm/wasi_llm.cpp:34: Failure
Expected: (LLMMod) != (nullptr), actual: NULL vs (nullptr)
[1]    1953366 segmentation fault (core dumped)  build/test/plugins/wasi_llm/WasiLLMTests
```

Reply:
I am wondering the output of running `WasiLLMTests` inside this folder `build/test/plugins/wasi_llm`.

Reply:
> I am wondering the output of running `WasiLLMTests` inside this folder `build/test/plugins/wasi_llm`.

ahh, that works.

Reply:
I think it's the relative path. And using `ctest` on the root folder will also work.